db/comdb2.c

```diff
@@ -4589,6 +4589,16 @@ void *statthd(void *p)
                                     last_bdb_stats.pwrite_time_us) /
                                     npwrites);
                 }
+                if (cur_bdb_stats.n_memp_fgets > last_bdb_stats.n_memp_fgets) {
+                    unsigned n_memp_fgets = cur_bdb_stats.n_memp_fgets - last_bdb_stats.n_memp_fgets;
+                    unsigned us = cur_bdb_stats.memp_fget_time_us - last_bdb_stats.memp_fget_time_us;
+                    reqlog_logf(statlogger, REQL_INFO, "n_memp_fgets=%u, avg time=%ums\n", U2M(us) / n_memp_fgets);
```

Comment:
The format has 2 arguments, but only 1 is provided? Also maybe rename avg time to something else to not confuse with the one above.

Reply:
Fixing (leftover from other branch). Thanks.